client/metric_dashboards.go

```diff
@@ -67,6 +68,15 @@ type Panel struct {
 	Body     map[string]any  `json:"body"`
 }
 
+type Threshold struct {
+	// enum: E,GE,GT,LE,LT
+	Operator string  `json:"operator"`
+	Value    float64 `json:"value"`
+	// An alpha hex color
+	Color string  `json:"color"`
+	Label *string `json:"label"`
```

Comment:
re: https://github.com/lightstep/terraform-provider-lightstep/pull/231/files#r1651587958
```suggestion
	Label string `json:"label"`
```